modules/osdk-deploy-olm.adoc

```diff
@@ -40,22 +40,29 @@ $ operator-sdk olm status \
     --olm-namespace=openshift-operator-lifecycle-manager
 ----
 
-. Run the Operator on your cluster by using the OLM integration in Operator SDK:
+. Enter the following command to run the Operator on the cluster: 
 +
 [source,terminal]
 ----
-$ operator-sdk run bundle \
-    [-n <namespace>] \//<1>
-    <registry>/<user>/<bundle_image_name>:<tag>
+$ operator-sdk run bundle \//<1>
+    [-n <namespace>] \//<2>
```

Comment:
This is more of an observation, but I've never seen -n <namespace> in brackets, and I'm not sure if it would work with brackets (in the past I've only used it as, for example, "$ operator-sdk run bundle -n <namespace> <registry>/<user>/<bundle_image_name>:<tag>".

I know it passed QE. Maybe it does work with brackets?

Reply:
I added an `Optional:` at the start of the callout and removed the brackets to be more comformant with our practices.